federation-v1/gateways/mesh-bun/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM oven/bun:1.0.15
+FROM oven/bun:latest
```

Comment:
We don't use `latest` on purpose for Bun to let Renovate update it to avoid unexpected failures.

Reply:
ah good point, reverting it

Reply:
Updated `mesh-supergraph-bun` to `1.0.26` and worked locally. Rolled back `mesh-bun` to current version.